pkg/codegen/docs/bundler.go

```diff
@@ -96,12 +96,16 @@ func main() {
 	contents := make(map[string][]byte)
 	for _, f := range files {
 		if f.IsDir() {
-			fmt.Printf("%q is a dir. Skipping...", f.Name())
+			fmt.Println(fmt.Sprintf("%q is a dir. Skipping...", f.Name()))
```

Comment:
Nit: Why not the following?

```suggestion
			fmt.Printf("%q is a dir. Skipping...\n", f.Name())
```

We do that pretty pervasively in the CLI.

Reply:
Only because I preferred this way over writing strings with new-line chars. I will change this to use `\n` instead of the enveloped call. I don't have strong ties with the current format.